packages/provider.component/spec/customElementBehaviors.ts

```diff
@@ -449,9 +459,17 @@ describe('Components: Custom elements', function () {
   })
 
   it('Disposes the component when the custom element is cleaned', function () {
-        // This is really a behavior of the component binding, not custom elements.
-        // This spec just shows that custom elements don't break it for any reason.
-    var componentViewModel = {
+
+
+    interface myModel {
```

Comment:
I have convert this setup to a class, see https://github.com/Auge19/tko/pull/4/commits/81980b774192ecaa91515878330a7a4c0c060ab4